apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -67,49 +70,62 @@ defmodule SiteWeb.ScheduleController.TimetableController do
     end
   end
 
-  @spec trip_messages(Routes.Route.t(), 0 | 1) :: %{{String.t(), String.t()} => String.t()}
-  defp trip_messages(%Routes.Route{id: "CR-Haverhill"}, 0) do
+  @spec trip_messages(Routes.Route.t(), 0 | 1, Date.t()) :: %{
```

Comment:
Is `Date.t()` the same as `Calendar.date()`?

Reply:
https://hexdocs.pm/elixir/Calendar.html seems to be an interoperability API.